web/src/main/webapp/resources/partials/forms/update_hunter.html

```diff
@@ -0,0 +1,61 @@
+<form name="form"  novalidate class="form-horizontal">
+
+    <div class="form-group" ng-class="{'has-error': !(form.firstName.$valid)}">
+        <label for="firstName" class="col-sm-2 control-label">First Name</label>
+        <div class="col-sm-10">
+            <input placeholder="firstName" id="firstName" type="text" ng-minlength="3" ng-maxlength="50" name="firstName" class="form-control" ng-model="hunter.firstName" />
+            <p class="help-block" ng-show="form.firstName.$error.minlength">First name must be at least 3 characters</p>
+            <p class="help-block" ng-show="form.firstName.$error.maxlength">First name must be at shorter than 50 characters</p>
+        </div>
+    </div>
+
+    <div class="form-group" ng-class="{'has-error': !(form.surname.$valid)}">
+        <label for="surname" class="col-sm-2 control-label">Surname</label>
+        <div class="col-sm-10">
+            <input placeholder="surname" id="surname" type="text" ng-minlength="3" ng-maxlength="50" name="surname" class="form-control" ng-model="hunter.surname" />
+            <p class="help-block" ng-show="form.surname.$error.minlength">Surname must be at least 3 characters</p>
```

Comment:
I guess all of these constraints should be also defined on server. It is not really safe when you can call rest api to circumvent these checks